charts/jenkins/pod-templates/nuxeo-jsf-lts-2023.yaml.gotmpl

```diff
@@ -0,0 +1,32 @@
+- id: "nuxeo-jsf-lts-2023"
+  name: "nuxeo-jsf-lts-2023"
+  label: "jenkins-nuxeo-jsf-lts-2023"
+  imagePullSecrets:
+  - name: {{ .Values.podTemplates.imagePullSecretName }}
+  serviceAccount: {{ .Values.podTemplates.serviceAccount }}
+  nodeUsageMode:  {{ .Values.podTemplates.nodeUsageMode }}
+  containers:
+  {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
+  - args: ""
+    command: ""
+    image: "{{ .Values.docker.registry }}/nuxeo/builder-java11-firefox42-nodejs10:latest"
+    alwaysPullImage: true
+    name: "maven"
+    privileged: {{ .Values.podTemplates.privileged }}
+    resourceLimitCpu: "3"
+    resourceLimitMemory: "6Gi"
+    resourceRequestCpu: "1.5"
+    resourceRequestMemory: "4Gi"
+    ttyEnabled: {{ .Values.podTemplates.ttyEnabled }}
+    workingDir: {{ .Values.podTemplates.workingDir }}
+  envVars:
+  {{- toYaml .Values.podTemplates.envVars | nindent 2 -}}
+  {{- toYaml .Values.podTemplates.javaEnvVars | nindent 2 -}}
+  {{- toYaml .Values.podTemplates.mavenEnvVars | nindent 2 -}}
+  volumes:
+  {{- toYaml .Values.podTemplates.volumes | nindent 2 -}}
+  {{- toYaml .Values.podTemplates.mavenSettingsVolumes | nindent 2 -}}
+  nodeSelector: {{ .Values.podTemplates.nodeSelector }}
+  yamlMergeStrategy: {{ .Values.podTemplates.yamlMergeStrategy }}
+  yaml: |
+    {{- .Values.podTemplates.yaml | nindent 4 -}}
```

Comment:
I'm not sure if the absence of new line at the EOF is a problem. This is duplicated from 2021 templates